src/Annotations/Assertion.php

```diff
@@ -11,21 +11,28 @@
 
 use function is_array;
 use function ltrim;
+use function trigger_error;
+
+use const E_USER_DEPRECATED;
 
 /**
  * Use this annotation to validate a parameter for a query or mutation.
  *
+ * Note 1: using this attribute as a target to the method (not parameter) is deprecated and will be removed in 8.0.
+ * Note 2: support for `doctrine/annotations` will be removed in 8.0.
+ * Note 3: this class won't implement `ParameterAnnotationInterface` in 8.0.
+ *
  * @Annotation
  * @Target({"METHOD"})
  * @Attributes({
  *   @Attribute("for", type = "string"),
  *   @Attribute("constraint", type = "Symfony\Component\Validator\Constraint[]|Symfony\Component\Validator\Constraint")
  * })
```

Comment:
Can be deleted as not working in v8 I assume

Reply:
Ahhh, as long as you are targetting to v7.1 then it can be left for those using v7 of the graphqlite. Sorry, disregard the comment above